keras/backend/cntk_backend.py

```diff
@@ -1111,6 +1111,10 @@ def less_equal(x, y):
     return C.less_equal(x, y)
 
 
+def where(condition, x, y):
+    raise NotImplementedError('CNTK not support ops like `tf.where`.')
```

Comment:
@souptc is there any CNTK equivalent we could be using here? Thanks!

Reply:
I don't think so, adding @KeDengMS from cntk team to confirm

Reply:
You may use [element_select](https://cntk.ai/pythondocs/cntk.ops.html#cntk.ops.element_select)

Reply:
@KeDengMS @souptc Thanks~